server/README.md

```diff
@@ -65,6 +65,19 @@ And then each host controller can be started as:
 
 The Server Management Console listens on the domain controller on port 9990. Credentials are admin/admin.
 
+## Source to image (S2I)
+
+Infinispan Docker image uses S2I to supply configuration XML file to the server. The scripts copy content of user directory
+into `/opt/jboss/infinispan-server/standalone/configuration`. The destination directory can be changed using `CONFIGURATION_PATH`
+environmental variable.
+
+The easiest way to run Infinispan with custom configuration inside OpenShift is to invoke the following command:
+
+    oc new-app jboss/infinispan-server~https://github.com/<username or organization>/<repository with xml in its root>.git
```

Comment:
If we provided a default clustered.xml file, it will not work I suppose, since it needs the kube_ping stack configured and the namespace. Should we mention this in the doc?